### PR TITLE
CORE-660: added the job_limits table

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/main/data/98_job_limits.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Populates the job_limits table.
+
+INSERT INTO job_limits (id, launcher, concurrent_jobs) VALUES
+    ('72DFEF9A-F6B1-482E-B2A0-16194247BE31', NULL, 8);
```
Comment (slr71) [inline src/main/data/98_job_limits.sql:4]: The row with the `NULL` launcher contains the default limits.

Comment (psarando) [reply src/main/data/98_job_limits.sql:4]: What is `launcher` for? Is it a username?

**Review by psarando — APPROVED** [review]
LGTM 👍

File: src/main/data/98_job_limits.sql (hunk 1 above)
Comment (slr71) [reply src/main/data/98_job_limits.sql:4]: Yeah, it's either a username or, eventually a group name. There's a feature request to allow users to launch analyses on behalf of a group that they belong to.

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**